cli/check.translation.php

```diff
@@ -199,7 +199,7 @@ function findUsedTranslations(): array {
 	$iterator = new RecursiveIteratorIterator($directory);
 	$regex = new RegexIterator($iterator, '/^.+\.(php|phtml)$/i', RecursiveRegexIterator::GET_MATCH);
 	$usedI18n = [];
-	foreach (array_keys(iterator_to_array($regex)) as $file) {
+	foreach ($regex as $file => $value) {
```

Comment:
@aledeg Please double-check.

In this PR, I have tried to remove some of the `iterator_to_array()` constructs